nix/cardano-services/deployments/default.nix

```
@@ -445,6 +445,8 @@ in
             chain-history-provider = {
               enabled = true;
               replicas = 2;
+              resources.limits = mkPodResources "500Mi" "2000m";
+              resources.requests = mkPodResources "150Mi" "1000m";
```

I think you need `mkForce` for this. Also has this been tested? We might need to remove limit altogether to resolve this issue

---

Probably we will end up with something like this. Also please check grafana memory usage, 150m feels low
```suggestion
              resources = lib.mkForce {
                requests = mkPodResources "150Mi" "1000m";
              };
```